edgelet/edgelet-docker/Cargo.toml

```diff
@@ -17,7 +17,9 @@ log = "0.4"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
+sysinfo = "0.9.5"
```

Comment:
Make this `0.9.6` so that it's clear we have to use the version with the bugfix, and so that nobody can accidentally downgrade it later.